assets/app/views/project.html

```diff
@@ -7,11 +7,20 @@
       </div>
       <!-- TODO handle things that don't live under services -->
       <alerts alerts="alerts"></alerts>
-      <div ng-if="(services | hashSize) == 0">
-        <div>
-          <em>{{emptyMessage}}</em>
-        </div>
+      <div ng-if="loadingServices">Loading...</div>
```

Comment:
@liggitt There's a different message when all projects are hidden by filters, see updateFilterWarning(). But you're right that I'm incorrectly showing get started as well. Good catch. I'll fix.